packages/cfd/src/Containers/jurisdiction-modal/__test__/jurisdiction-card-section.spec.tsx

```diff
@@ -90,7 +90,7 @@ describe('JurisdictionCardSection', () => {
         },
         type_of_card: Jurisdiction.SVG,
         toggleCardFlip: jest.fn(),
-        verification_docs: ['not_applicable'],
+        verification_docs: [],
```

Comment:
so are we removing "not_applicable" status @shontzu-deriv

Reply:
if I can get a green light from you 🙏🏼

Reply:
yes @amina-deriv

Reply:
reason for that is we don't want this (icon component is being rendered with no icon):

<img width="600" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/3c91dddd-95b1-4cfc-b798-69e754329333">

Reply:
reason for that is we don't want this (icon component is being rendered with no icon) which is happening in the previous commit:

<img width="600" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/3c91dddd-95b1-4cfc-b798-69e754329333">